.travis.yml

```diff
@@ -23,59 +23,56 @@ matrix:
     - python: "2.7"
       env:
         - TOX_ENV=pythonbuild2.7
-        - TRAVIS_NODE_VERSION="6"
```

Comment:
Just typed in '6' as the version in the one particular place where NPM is installed :)

Reply:
that sounds very nice!